pkg/reporter/junit_reporter.go

```diff
@@ -198,10 +198,14 @@ func (jr JunitReporter) Print(reports []Report) error {
 	}
 
 	results := Header + string(data)
-	fmt.Println(results)
 
 	if jr.outputDest != "" {
 		return outputBytesToFile(jr.outputDest, "result", "xml", []byte(results))
 	}
+
+	if len(reports) > 0 && !reports[0].IsQuiet {
```

Comment:
What if the second reporter is quiet?

Reply:
Isn't the quiet flag global? Like it applies to all reporters specified as args.

Otherwise might have to add some sort of logic to enable printing from let's say the last reporter?

Reply:
I don't know, I was also surprised the quiet flag wasn't global.

Maybe it requires a refactoring.

Another point of view may help. @kehoecj maybe

Reply:
I just ran the validator from your branch and reporting seems to be working as expected with the quiet flag. Here's the test cases I ran. 

1. Quiet flag with a json reporter. No stdout as expected
    ```
    docker run --rm cfv:multi --quiet --reporter json
    ```
2. Quiet flag with a json reporter using the `:-` file output. No stdout as expected
    ```
    docker run --rm cfv:multi --quiet --reporter json:-
    ```
3. Quiet flag with a json reporter and specifying ` file output. No stdout as expected AND the `scan.json` output file is populated with the report output
    ```
    docker run --rm cfv:multi --quiet --reporter json:/tests/scan.json
    ```
4. Quiet flag with every report type and specifying file output. No stdout as expected AND all files are properly populated
    ```
    docker run --rm cfv:multi --quiet --reporter json:/tests/scan.json --reporter junit:/tests/scan.xml --reporter=sarif:/tests/scan.sarif --reporter=standard:/tests/scan.txt
    ```
5. Quiet flag with every report type without output file. No stdout as expected.
    ```
    docker run --rm cfv:multi --quiet --reporter json--reporter junit --reporter=sarif --reporter=standard
    ```

Really great work on this @xanish

Reply:
thanks bro @kehoecj :smile: 

fixing up the 2 minor issues pointed and will push in sometime